0-bootstrap/README.md

```diff
@@ -10,7 +10,7 @@ the example.com reference architecture described in
 <td>0-bootstrap (this file)</td>
 <td>Bootstraps a Google Cloud organization, creating all the required resources
 and permissions to start using the Cloud Foundation Toolkit (CFT). This
-step also configures a CI/CD pipeline for foundations code in subsequent
+step also configures a Foundation Pipeline for foundations code in subsequent
```

Comment:
Can we rename this to `Foundation CI/CD pipeline` in glossary, keep CI/CD pipeline here and add a link instead? Reasoning is that most users maybe familiar with CI/CD but `Foundation Pipeline` alone may confuse them having to click through only to realize it is a ci/cd pipeline.

Reply:
Done